guides/common/modules/proc_remediating-compliance-failures.adoc

```diff
@@ -10,24 +11,30 @@ Remediation might render the system non-functional.
 ====
 
 .Prerequisites
-* Your user account has a role assigned that has the `view_arf_reports` and `view_hosts` permissions.
+* Your user account has a role assigned that has the following permissions: `view_arf_reports`, `view_hosts`, `create_job_invocations`
 
 .Procedure
-To examine a simplified compliance policy report:
+To examine a simplified compliance policy report and run the compliance failure remediation wizard:
 
 . In the {ProjectWebUI}, navigate to *Hosts* > *Compliance* > *Reports*.
 . In the *Reported At* column, click the time link of the report you want to examine.
-{Project} displays a simplified list of policy rules with the results of the scan.
+{Project} displays a list of log messages describing the results of the scan.
+. Locate a log message that describes a failed compliance check.
+In the *Actions* column, select *Remediation* to open the compliance remediation wizard.
+Follow the wizard to remediate the compliance failure.
++
+The remediation wizard might not be available for all compliance failures.
```

Comment:
```suggestion
The remediation wizard might not be available for all compliance failures.
```
Maybe this can be a note?

Reply:
Ok, I'll turn this and the other one into a note.